pages/index.html

```diff
@@ -44,107 +44,93 @@
         </script>
     </head>
 
-    <body id="search" class="CoveoSearchInterface" data-enable-history="true" style="padding: 1em;">
-        <span class="CoveoAnalytics"></span>
+    <body id="search" class="CoveoSearchInterface" data-enable-history="true">
         <div class="coveo-tab-section">
             <a class="CoveoTab" data-id="All" data-caption="All Content"></a>
         </div>
         <div class="coveo-search-section">
-            <div class="CoveoSettings"></div>
             <div class="CoveoSearchbox" data-enable-omnibox="true"></div>
+            <div class="CoveoSettings"></div>
         </div>
         <div class="coveo-main-section">
             <div class="coveo-facet-column">
-                <div class="CoveoAttachedResultsFilter"></div>
                 <div class="CoveoFacet" data-title="Type" data-field="@objecttype" data-tab="All"></div>
                 <div class="CoveoFacet" data-title="FileType" data-field="@filetype" data-tab="All"></div>
                 <div class="CoveoFacet" data-title="Author" data-field="@author" data-tab="All"></div>
-                <div class="CoveoFacet" data-title="Year" data-field="@year" data-tab="All"></div>
-                <div class="CoveoFacet" data-title="Month" data-field="@month" data-tab="All"></div>
-            </div>
-            <div class="coveo-search-section">
-                <div class="CoveoSettings"></div>
-                <div class="CoveoSearchbox" data-enable-omnibox="true"></div>
+                <div class="CoveoTimespanFacet" data-tab="All"></div>
             </div>
-            <div class="coveo-main-section">
-                <div class="coveo-facet-column">
-                    <div class="CoveoResultsFilter" data-text="Show Only Attached Results"></div>
-                    <div class="CoveoFacet" data-title="Type" data-field="@objecttype" data-tab="All"></div>
-                    <div class="CoveoFacet" data-title="FileType" data-field="@filetype" data-tab="All"></div>
-                    <div class="CoveoFacet" data-title="Author" data-field="@author" data-tab="All"></div>
-                    <div class="CoveoFacet" data-title="Year" data-field="@year" data-tab="All"></div>
-                    <div class="CoveoFacet" data-title="Month" data-field="@month" data-tab="All"></div>
+            <div class="coveo-results-column">
+                <div class="CoveoShareQuery"></div>
```

Comment:
Is this page supposed to mimic a side panel? if so, you could probably remove the CoveoSettings, CoveoShareQuery, CoveoExportToExcel, CoveoPreferencesPanel etc.

If it's a normal page, disregard.

Reply:
It's a normal page, based on the JSUI one. The only changes that I've done there is to remove duplicated main-section (This comment apply mostly to every of you comment on this file I think)